bindings_ffi/src/mls.rs

```diff
@@ -7787,6 +7812,37 @@ mod tests {
         assert_eq!(final_alix_messages.len(), 5, "Alix should see 5 messages");
     }
 
+    #[tokio::test(flavor = "multi_thread", worker_threads = 5)]
+    async fn test_can_add_installation_with_a_bad_key_package() {
```

Comment:
I think we should delete this test before we merge it

Reply:
Before I removed this test I also ran it with the history sync server enabled and everything worked as expected. 🎉

Reply:
the test was flaky due to issue of kps order. now it should be safe to have the tests back.